### ISSUE TITLE
Missing alpha blended vectors

### ISSUE THREAD

**Issue by midzer** [ptitSeb/gl4es#472]
Hi all,

I am in the process of porting https://github.com/mdsteele/azimuth for the web using Emscripten. The game uses OpenGL 2.1 functions, so I need gl4es as translation library.

I've replaced

```
#include <SDL_opengl.h>
```

with

```
#include <GL/gl.h>
```

and loaded `initialize_gl4es()` early on.

The compile runs fine, but there are certain (later destructable) alpha blended wall structures which are not visible in the port. In the native build those wall structures are visible.

I already tried to tweak some environment variables via https://github.com/ptitSeb/gl4es/blob/master/USAGE.md without success.

Thanks for your support and have a nice week
midzer

**Comment by ptitSeb** [comment]
Do you have any idea what kind of drawing those wall are doing?

**Comment by midzer** [comment]
Without deep OpenGL knowledge, I guess it has to be somewhere around https://github.com/mdsteele/azimuth/blob/050f838b35d19ffdc738f33178abaf9d69d834ec/src/azimuth/view/wall.c#L492

**Comment by midzer** [comment]
The original author answered https://github.com/mdsteele/azimuth/issues/25#issuecomment-2405183156

**Comment by midzer** [comment]
This error is also thrown after level is loaded where those walls are opaque:

> GL_INVALID_OPERATION: Vertex buffer is not big enough for the draw call

**Comment by midzer** [comment]
In addition I had to exchange 3rd <-> 4th vertex.

**Comment by ptitSeb** [comment]
Yeah, gl4es transformation quads->strip should be correct. But I guess there is an attribute somehow that got lost in the transformation.

**Comment by midzer** [comment]
Don't know.

Here are all the changes https://github.com/midzer/azimuth/tree/replaceQuads
